### DISCUSSION TITLE
Is the order of suggest params influnce the result?

### DISCUSSION THREAD

**Discussion by fengshansi** [optuna/optuna#5590] [Q&A]
I changed the order of suggest params and get different results.

**Comment by THB0705** [comment]
Your question is not very clear. How are you changing the order of params?

One way in which the order of suggestions might result in different values is because the seed number for particular params are not being fixed. 

I am still a beginner at Optuna. However, I believe that if you were to run the same experiment twice you will get different results. This is because the seed number is not being fixed. This results in different parameters being suggested and it could be that a different minimum (or maximum) objective might be found. On top of this you are changing the order of your parameter suggestions between experiments. This probably would make reproducing parameter suggestions even more tricky.

**Reply by fengshansi** [reply]
> 你的问题不是很清楚。你如何改变参数的顺序？
> 
> 建议的顺序可能导致不同值的一种方式是，特定参数的种子编号未固定。
> 
> 我仍然是 Optuna 的初学者。但是，我相信，如果你要运行两次相同的实验，你会得到不同的结果。这是因为种子编号不是固定的。这导致建议不同的参数，并且可能会找到不同的最小（或最大）目标。除此之外，您还需要更改实验之间参数建议的顺序。这可能会使重现参数建议变得更加棘手。

I set the same seed. The order means if I have two params A and B, I suggest A first and suggest B first get different result. But they can both be repeated because I set the seed.
